internal/accounts/account-add.go

```diff
@@ -0,0 +1,88 @@
+package accounts
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/spf13/cobra"
+)
+
+type flagsAdd struct {
+	KeySigAlgo  string `default:"ECDSA_P256" flag:"sig-algo" info:"Signature algorithm"`
+	KeyHashAlgo string `default:"SHA3_256" flag:"hash-algo" info:"Hashing algorithm"`
+	KeyIndex    int    `flag:"index" info:"Account key index"`
+	KeyHex      string `flag:"privateKey" info:"Private key in hex format"`
+	KeyContext  string `flag:"context" info:"Projects/<PROJECTID>/locations/<LOCATION>/keyRings/<KEYRINGID>/cryptoKeys/<KEYID>/cryptoKeyVersions/<KEYVERSION>"`
```

Comment:
```suggestion
```

I really think we should take this out (just from the CLI command, it can stay in `pkg/flow`).

It's just that I'm pretty sure we'll want to change this in the future and it'd be nice to avoid a breaking change. Anybody blocked by this can just create the account manually in `flow.json`